front/src/components/UI/AuctionView/AuctionBlock.tsx

```diff
@@ -0,0 +1,20 @@
+import { AuctionViewStyle } from "./AuctionView.styled";
+import { AuctionViewText } from "./AuctionView.styled";
+import { ProductProps } from "../ProductView/ProductBlock";
+import { AuctionViewTitle } from "./AuctionView.styled";
+
+export interface AuctionProps extends ProductProps {
```

Comment:
mimo wszytsko lepiej bez dziedziczenia zapisać. Te elementy się raczej komponują niż dziedziczą po sobie. Bo co w sytuacji gdy produkt będzie miał jakiś parametr który dla aukcji jest bezużyteczny albo nawet nieprawidłowy? A jak są rozdzielone to nie zależą od siebie bezpośrednio